substrate/client/transaction-pool/tests/zombienet/small_network_yap.rs

```diff
@@ -0,0 +1,142 @@
+use zombienet_sdk::{
+	LocalFileSystem, Network as ZNetwork, NetworkConfig, NetworkConfigBuilder, NetworkConfigExt,
+};
+
+use super::{
+	Network, Node, ParachainConfig, RelaychainConfig, DEFAULT_PC_NODE_RPC_PORT,
+	DEFAULT_RC_NODE_RPC_PORT,
+};
+use anyhow::anyhow;
+use which::which;
+
+// A zombienet network with two relaychain 'polkadot' validators and one parachain
+// validator based on yap-westend-live-2022 chain spec.
+pub struct SmallNetworkYap {
+	rc_config: RelaychainConfig,
+	pc_config: ParachainConfig,
+	rc_nodes: Vec<Node>,
+	pc_nodes: Vec<Node>,
+}
+
+impl SmallNetworkYap {
+	/// Creates a new [`SmallNetworkYap`].
+	pub fn new(rc_config: RelaychainConfig, pc_config: ParachainConfig) -> Self {
+		SmallNetworkYap {
+			rc_config,
+			pc_config,
+			rc_nodes: vec![
+				Node::new("alice".to_owned(), vec![]),
+				Node::new("bob".to_owned(), vec![]),
+			],
+			pc_nodes: vec![Node::new(
+				"charlie".to_owned(),
+				vec![
+					"--force-authoring".into(),
+					("--pool-limit", "500000").into(),
```

Comment:
dq: Would we want to configure this in the future?

One option would be to add a tiny `builder` wrapper around this struct, then we could image some scenarios like:

```rust
let small_net: SmallNetworkYap = SmallNetworkYapBuilder::default()
  .with_pool_limit(5k)
  .with_pool_kbytes(204)
  .build(relay_chain, para_chain);

let small_net_2: SmallNetworkYap = SmallNetworkYapBuilder::default()
  .with_pool_limit(1k)
  .with_pool_kbytes(1k)
  .build(relay_chain, para_chain);
```

Could be a followup, feel free to ignore me :D

Reply:
Very good point. Down the line we should enable the arg list to be passed according to the substrate-cli. I'll rethink this part a bit. We should consider that there is already a default arg list, and maybe we can use them in a dedicated constructor that would include them, but also provide a constructor version with no args, for custom initialization.